lib/portlayer/storage/nfs/volume.go

```diff
@@ -107,6 +107,7 @@ func (v *VolumeStore) VolumeCreate(op trace.Operation, ID string, store *url.URL
 	}
 
 	u, _ := v.Service.URL()
+	u.Scheme = "nfs"
```

Comment:
yeah I felt that way as well. I will look at it in just a moment. I wanted to hand test and forget to inspect this more.